### ISSUE TITLE
Problem with gradle

### ISSUE THREAD

**Issue by ingouackaz** [skiptools/skipapp#1]
Hello ! 

When I try to compile your example project, I have this error :

` Error: /Users/myaccount/Library/Developer/Xcode/DerivedData/SkipWeather-grnbjezbqysvqsfsdeonunuojuoe/Build/Intermediates.noindex/SkipWeather.build/Debug-iphonesimulator/SkipWeather.build/skip-gradle.log.txt:26:0: error: The gradle command failed. Review the log for details and consult https://skip.tools/docs/faq for common solutions. Command: gradle -p ../Android launchDebug `

```
* Where:
Settings file '/Users/myaccount/Downloads/skipapp-weather-main/Android/settings.gradle.kts' line: 38

* What went wrong:
An exception occurred applying plugin request [id: 'skip-plugin']
> Failed to apply plugin 'skip-plugin'.
   > Could not read script '/Users/myaccount/Library/Developer/Xcode/DerivedData/SkipWeather-grnbjezbqysvqsfsdeonunuojuoe/SourcePackages/plugins/skipapp-weather.output/SkipWeather/skipstone/settings.gradle.kts' as it does not exist.
```

Do you know how I can fix it ?

**Comment by marcprux** [comment]
That's curious. Is this a brand-new project, or was it created a while ago? Also, are you able to run `skip checkup` withput problem?

**Comment by ingouackaz** [comment]
Yes ! It works with this command ` skip init --appid=bundle.id project-name AppName` but

**Comment by ingouackaz** [comment]
Yes @marcprux it works with this `skip init --appid=bundle.id project-name AppName`

**Comment by marcprux** [comment]
Can you try manually deleting the `/Users/myaccount/Library/Developer/Xcode/DerivedData/SkipWeather-grnbjezbqysvqsfsdeonunuojuoe/SourcePackages/plugins/skipapp-weather.output` folder and trying to build again? Let us know if the same error occurs (or a different one). FTR, I am able to run the `skipapp-weather` app locally, so it might just have been an Xcode hiccup.
